package.json

```diff
@@ -13,13 +13,14 @@
         "postinstall": "cd client && yarn"
     },
     "dependencies": {
-        "@nimiq/keyguard-client": "0.1.6",
-        "@nimiq/network-client": "^0.1.3",
+        "@nimiq/network-client": "^0.1.5-beta2",
```

Comment:
Downgrade of network-client? Is this correct?

Reply:
How is that a downgrade?

Reply:
Ah, I was confused, because you changed the order of the dependencies. Please fix the order. keyguard-client comes before network-client in the alphabet.